src/kimmdy/runmanager.py

```diff
@@ -398,6 +398,16 @@ def _setup(self, files: TaskFiles) -> TaskFiles:
         write_top(self.top.to_dict(), files.outputdir / self.config.top.name)
         files.output["top"] = files.outputdir / self.config.top.name
         logger.info("Done with setup")
+
+        # copy input files that are potentially modified
+        # to the setup task directory
+        # by applying a recipe (e.g. by trunkate_sim_files)
+        for f in ["xtc", "tpr", "trr"]:
```

Comment:
This can be a very expensive operation!
It might be better to prevent the truncation function from working in directories not created by this run.

Reply:
yeah, I'm still not settles on how to ultimately solve this. Because it if doesn't truncate, the application of the recipe may produce non-sensible trajectories.

Reply:
not copying for now, solution is something for a new PR